config/kubernetes/sig-release/teams.yaml

```
@@ -43,6 +43,7 @@ teams:
     - cantbewong # VMware
     - caseydavenport # Network
     - cheftako # Cloud Provider
+    - chandanim123 # 1.21 Docs Shadow
```

```suggestion
    + ChandaniM123 # 1.21 Docs Shadow
```

https://github.com/kubernetes/org/issues/2436 will be closed today via https://github.com/kubernetes/org/pull/2441

---

Also, please swap   
   - cheftako # Cloud Provider
   - chandanim123 # 1.21 Docs Shadow